demos/supabase/.env.template

```diff
@@ -1,8 +1,7 @@
 # ==================== Supabase config ============================
-# Get these after executing `supabase start`
+# Get these after executing `supabase start` (or ./setup.sh)
```

Comment:
We might be able to provide a default value for the publishable secret. I've noticed before that the JWT secret seemed to the same value between multiple local Supabase projects. I tested this with the publishable secret and also got the same value of `sb_publishable_ACJWlzQHlZjBrEguHvfOxg_3BJgxAaH` for 2 projects. 

Seems like there are defaults https://github.com/supabase/cli/blob/7406feeff85cda1f0a13bcb22e2f8ab57dd277aa/pkg/config/apikeys.go#L19